virtual-desktop/src/app/window-manager/simple-window-manager/simple-window-manager.service.ts

```diff
@@ -93,6 +93,7 @@ export class SimpleWindowManagerService implements MVDWindowManagement.WindowMan
   private generateSessionEventsProvider(): Angular2PluginSessionEvents {
     const loginSubject = new Subject<void>();
     const sessionExpireSubject = new Subject<void>();
+    const autosaveEmitter = new Subject<any>();
```

Comment:
Because this is never used, it means single app mode will not have autosave. I think it should, and that can be done next. In single app mode, you would restore data from index 0 of the app requested. All other apps and all other instances of that app would be ignored.

Reply:
Done AutoSaving for Single App Mode